.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: ./scripts/deps.sh
+        run: sudo apt-get update && ./scripts/deps.sh
```

Comment:
Should `deps.sh` just do this itself?

Reply:
Yes, I think so

Reply:
Don't see any reason not to; updated.

Apparently `dnf install` already updates it's cache, so nothing needs to be changed for it.